CHANGELOG.md

```diff
@@ -6,6 +6,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [12.0.0] - 2021-07-07
+### Added
+- Added GasFeeController to provide gas fee estimates [#494](https://github.com/MetaMask/controllers/pull/494)
+
+### Changed
+- **BREAKING**: Added chainId support to TokenRatesControllers [#476](https://github.com/MetaMask/controllers/pull/476)
+  - The breaking change here is that TokenRatesController constructor now requires a onNetworkStateChange listener
+- Added iconUrl to Token type in TokenListController [#512](https://github.com/MetaMask/controllers/pull/512)
```

Comment:
Nit: Our change entries are imperative rather than past-tense, just as is advised for git commit messages (the commit _is_ the change).
```suggestion
- Add GasFeeController to provide gas fee estimates [#494](https://github.com/MetaMask/controllers/pull/494)

### Changed
- **BREAKING**: Add chainId support to TokenRatesControllers [#476](https://github.com/MetaMask/controllers/pull/476)
  - The breaking change here is that TokenRatesController constructor now requires a onNetworkStateChange listener
- Add iconUrl to Token type in TokenListController [#512](https://github.com/MetaMask/controllers/pull/512)
```